onnxmltools/convert/keras/shape_calculators/GlobalPooling.py

```diff
@@ -0,0 +1,23 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+import keras
+from ...common._registration import register_shape_calculator
+
+def calculate_keras_global_pooling_output_shapes(operator):
+    d_len = len(operator.raw_operator.input_shape)
+    d_app = [1 for i in range(d_len -2)]
```

Comment:
`-2` -->`- 2`. Btw, does Keras not compute output shape correctly?

Reply:
Keras output shape is different from onnx spec.

Reply:
This is converter! We cannot change the output shape here. Otherwise, the downstream operators may get a unexpected shape.